source/common/http/utility.h

```diff
@@ -145,6 +145,18 @@ Http2Settings parseHttp2Settings(const envoy::api::v2::core::Http2ProtocolOption
  */
 Http1Settings parseHttp1Settings(const envoy::api::v2::core::Http1ProtocolOptions& config);
 
+struct LocalReplyInfo {
+  bool is_grpc{false};
+  bool is_head_request{false};
+  bool is_json_content_type{false};
+};
+
+/**
+ * @return LocalReplyInfo An LocalReplyInfo populated from the
```

Comment:
s/An LocalReplyInfo populated from the/A populated LocalReplyInfo object from a/